docs/mobile-apps/automated-testing/espresso-xcuitest/xcuitest.md

```diff
@@ -684,6 +684,26 @@ Setting `0` reverts to the value set in `defaults`.
 
 ---
 
+### `passThreshold`
+
+<p><small>| OPTIONAL | INTEGER |</small></p>
+
+Specifies the minimum number of successful attempts for a suite to be considered as `passed`. It should be used along with [retries](#retries).
+
+:::note
+For example, setting `retries` to 3 and `passThreshold` to 2.
+The max attempt would be 4 times. If the test passed twice, it'd stop and be marked as `passed`. Otherwise, it'd be marked as `failed`.
```

Comment:
```suggestion
The max attempt would be 4 times. If the test passed twice, it would stop and be marked as `passed`. Otherwise, it would be marked as `failed`.
```